packages/framework/attributor/src/attributor.ts

```diff
@@ -88,8 +88,8 @@ export class OpStreamAttributor extends Attributor implements IAttributor {
 	) {
 		super(initialEntries);
 		deltaManager.on("op", (message: ISequencedDocumentMessage) => {
-			const client = audience.getMember(message.clientId);
-			if (message.type === "op") {
+			if (message.clientId !== null && message.type === "op") {
```

Comment:
This could probably use a comment to explain the thinking behind the conditional.  I think it's probably something like:
`// We only want to perform attribution for messages that were sent by clients (not the server).  We also only want to attribute for messages that are likely to modify data, and messages with type "op" are a reasonable approximation of that.`